generic_tracers/generic_COBALT.F90

```diff
@@ -8070,7 +8070,8 @@ subroutine generic_COBALT_update_from_source(tracer_list,Temp,Salt,rho_dzt,dzt,h
 
     do k = 1, nk ; do j = jsc, jec ; do i = isc, iec   !{
        cobalt%jprod_lithdet(i,j,k)=( cobalt%total_filter_feeding(i,j,k)/ &
-                                   ( phyto(LARGE)%f_n(i,j,k) + phyto(DIAZO)%f_n(i,j,k) + epsln ) * &
+                                   ( phyto(LARGE)%f_n(i,j,k) + phyto(DIAZO)%f_n(i,j,k) + &
+                                     0.8*phyto(MEDIUM)%f_n(i,j,k) + 0.3*phyto(SMALL)%f_n(i,j,k) + epsln) * &
```

Comment:
@feida6996 @charliestock , Do you think there's a chance that we may need to retune these parameters (0.8, 0.3)? If so, should we consider putting them in the add_param [Remineralization section](https://github.com/NOAA-CEFI-Regional-Ocean-Modeling/ocean_BGC/blob/10613509b0c50400dc86c9b18bc5e92cc1a1a951/generic_tracers/generic_COBALT.F90#L5057-L5080)